0125000-trib-to-parsnip.Rmd

```diff
@@ -44,5 +114,47 @@ knitr::include_graphics(my_photo, dpi = NA)
 
 ```
 
+## Fish Sampling {.unnumbered}
+
+Electrofishing was conducted with results summarised in Tables
+\@ref(tab:tab-fish-site-125000) - \@ref(tab:tab-fish-dens-125000) and
+Figure \@ref(fig:plot-fish-box-125000).
+
+<br>
+
+## Conclusion {.unnumbered}
+
+Modelling indicates 
```

Comment:
there are no steelhead in this area due to arctic watershed and dams!! We will need to use bt for this program. Might not have modelling yet.